prospector.yml

```diff
@@ -26,11 +28,14 @@ pep8:
 
 pylint:
   options:
+    # Required for the Django plugin
+    django-settings-module: readthedocs.settings.test
     docstring-min-length: 20
     dummy-variables-rgx: '_$|__$|dummy'
     max-line-length: 100
   disable:
     # We do this for circular imports
+    - cyclic-import
```

Comment:
I can return with a triple-test of why this is needed and with the error that not disabling it causes.

It seems from the comment in the previous line that there has already been issues with circular imports. There are no changes in [the changeset](https://github.com/readthedocs/readthedocs.org/pull/10001) that explain why I'd get the issue.

I'm thinking it's not worth spending too much time with, but that it's better to go over linting tools and configuration at a higher level.

Reply:
Nope, check it again, it needs to stay, otherwise I get an issue that I don't know what to do about right now.

```
Messages
========

readthedocs/builds/models.py
  Line: 1
    pylint: cyclic-import / Cyclic import (readthedocs.builds.models -> readthedocs.projects.models)



Check Information
=================
         Started: 2023-03-16 17:27:43.477411
        Finished: 2023-03-16 17:27:58.640708
      Time Taken: 15.16 seconds
       Formatter: grouped
        Profiles: prospector.yml, full_pep8, doc_warnings, no_test_warnings, strictness_low, strictness_medium, strictness_high, strictness_veryhigh, no_member_warnings
      Strictness: from profile
  Libraries Used: django, celery
       Tools Run: dodgy, pep257, pep8, profile-validator, pyflakes, pylint
  Messages Found: 1
```